src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,167 @@
+package duke.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.tasklist.TaskList;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+/**
+ * Storage class to interact with the text file store in ./data.
+ */
+public class Storage {
+    private String cwd = System.getProperty("user.dir");
+    private java.nio.file.Path dataPath;
+    private String dataPathString;
+
+    /**
+     * Constructor.
+     * @param filePath the path to the text file that stores the serialised tasks
+     */
+    public Storage(String filePath) {
+        this.dataPath = java.nio.file.Paths.get(cwd, filePath.split("/"));
+        this.dataPathString = dataPath.toString();
+    }
+
+    /**
+     * Serialises task into a string that is storable in the text file.
+     * @param task the task to be serialise
+     * @return the string of the serialised task
+     */
+    private String serialise(Task task) {
+        String data = "";
+        String isDone = (task.getIsDone() ? "1" : "0");
```

Comment:
Maybe you want to rename this variable? Might be confusing as it is not actually a boolean.